qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -90,6 +105,115 @@ def quantum_instance(
         else:
             self._quantum_instance = quantum_instance
 
+    @property
+    def free_parameters(self) -> Union[ParameterVector, Sequence[Parameter]]:
+        """Return the vector of free parameters."""
+        return self._free_parameters
```

Comment:
We normally define instance vars in the constructor so the in an overview of the instance etc there - maybe initialized to None at that point if a value cannot be determined. I just commented against this one but there are others including the change in init() to set feature map via the setter rather than define/set the _feature_map var.